main/define/test/src/mill/define/ApplicativeTests.scala

```diff
@@ -28,7 +28,7 @@ object ApplicativeTests extends TestSuite {
       value
     }
   }
-  @compileTimeOnly("Target.ctx() can only be used with a T{...} block")
+  @compileTimeOnly("Target.ctx() can only be used with a Task {...} block")
```

Comment:
```suggestion
  @compileTimeOnly("Target.ctx() can only be used within a Task {...} block")
```

Reply:
This doesn't work yet; need to make changes in the mill-moduledefs repo. TBH I feel like just in-sourcing it again, since even when in-sourced people can depend on the artifact separately if necessary, and it makes doing these changes in one PR a lot more convenient

Reply:
Since it's a compiler module, it needs releases for every Scala version we want to use. That's the main reason we out-sourced it. We post-release for every new Scala 3.13 version.

It might be ok for a development version to use a in-sourced patched version, but I think the goal should be to push any changes back to the external project before we reach a stable version.

Reply:
How about we in-source it to the Mill repo but still keep the publishing separate? That would mean it's treated like the `bridge` modules, and would keep everything in one place while still preserving the per-scala-version releases and separate publishing

Reply:
That will work too, _if_ it's well documented. We do these things only occasionally, there is no muscle memory for the process, so we need to good step-by-step documentation targeting the developer/publisher.